src/modules/poweraccent/PowerAccent.Core/Languages.cs

```diff
@@ -0,0 +1,477 @@
+                    return new char[] { 'ź', 'ž', 'ż', 'ʒ', 'ǯ', 'ζ' };
+                case LetterKey.VK_COMMA:
+                    return new char[] { '¿', '¡', '∙', '₋', '⁻', '–', '≤', '≥', '≠', '≈', '≙', '±', '₊', '⁺', 'þ' };
+            }
+
+            return Array.Empty<char>();
+        }
+
+        // Currencies (source: https://www.eurochange.co.uk/travel-money/world-currency-abbreviations-symbols-and-codes-travel-money)
+        private static char[] GetDefaultLetterKeyCUR(LetterKey letter)
+        {
+            switch (letter)
+            {
+                case LetterKey.VK_B:
+                    return new char[] { '฿', 'в' };
+                case LetterKey.VK_C:
+                    return new char[] { '¢', '₡', 'č' };
+                case LetterKey.VK_D:
+                    return new char[] { '₫' };
+                case LetterKey.VK_E:
+                    return new char[] { '€' };
+                case LetterKey.VK_F:
+                    return new char[] { 'ƒ' };
+                case LetterKey.VK_H:
+                    return new char[] { '₴' };
+                case LetterKey.VK_K:
+                    return new char[] { '₭' };
+                case LetterKey.VK_L:
+                    return new char[] { 'ł' };
+                case LetterKey.VK_N:
+                    return new char[] { 'л' };
+                case LetterKey.VK_M:
+                    return new char[] { '₼' };
+                case LetterKey.VK_P:
+                    return new char[] { '£', '₽' };
+                case LetterKey.VK_R:
+                    return new char[] { '₹', '៛', '﷼' };
+                case LetterKey.VK_S:
+                    return new char[] { '$', '₪' };
+                case LetterKey.VK_T:
+                    return new char[] { '₮', '₺' };
+                case LetterKey.VK_W:
+                    return new char[] { '₩' };
+                case LetterKey.VK_Y:
+                    return new char[] { '¥' };
+                case LetterKey.VK_Z:
+                    return new char[] { 'z' };
+            }
+
+            return Array.Empty<char>();
+        }
+
+        // French
+        private static char[] GetDefaultLetterKeyFR(LetterKey letter)
+        {
+            switch (letter)
+            {
+                case LetterKey.VK_A:
+                    return new char[] { 'à', 'â', 'á', 'ä', 'ã' };
+                case LetterKey.VK_C:
+                    return new char[] { 'ç' };
+                case LetterKey.VK_E:
+                    return new char[] { 'é', 'è', 'ê', 'ë', '€' };
+                case LetterKey.VK_I:
+                    return new char[] { 'î', 'ï', 'í', 'ì' };
+                case LetterKey.VK_O:
+                    return new char[] { 'ô', 'ö', 'ó', 'ò', 'õ' };
+                case LetterKey.VK_U:
+                    return new char[] { 'û', 'ù', 'ü', 'ú' };
+                case LetterKey.VK_Y:
+                    return new char[] { 'ÿ', 'ý' };
```

Comment:
A lot of extra letters, some letters are missing. Here is what it should look like ([source](https://en.wikipedia.org/wiki/French_orthography)):

A: à, â, æ
C: ç
E: é, è, ê, ë
I: î, ï
O: ô, œ
U: ù, û, ü
Y: ÿ

Reply:
The extra letters are still there. For example, á, ä, ã, í, ó and many others in this list are not French.